views/peeps/index.erb

```diff
@@ -0,0 +1,10 @@
+<ul>
+  <% @peeps.each do |peep| %>
+    
+    <li>
+    <%= peep.peep %> 
+    peeped at  
```

Comment:
really loved this part, I wouldn't have thought to do this, but it's really nice for the user to see